Library/Homebrew/formula.rb

```diff
@@ -1271,7 +1271,7 @@ def to_hash
         "root_url" => bottle_spec.root_url,
       }
       bottle_info["files"] = {}
-      bottle_spec.collector.keys.each do |os|
+      bottle_spec.collector.each_key do |os|
```

Comment:
There’s indeed an issue that wasn’t caught in our tests: `bottle_spec.collector` isn’t a `Hash` here but a `BottleCollector` that defines `keys` but not `each_key`. I reverted this commit a couple minutes after merging this PR.